src/components/Categories.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Categories = () => (
+  <>
+    <p>Code is being implemented to this part soon!</p>
+  </>
```

Comment:
[OPTIONAL] As part of the project requirements, this component should contain a `<Button>` with the text "Check status". :wink: